DANS-KNAW/dd-pre-publish-workflow
# DD-334 & DD-336 Add lock override header to Dataverse API calls from prepublish workflow / execute from job queue

Fixes DD-334 and DD-336

# Description of changes
* Moved handling of workflow to separate task queue
* Corrected vault metadata logic, so that is now als works for imports

# How to test
* Make sure you are using a Dataverse that includes the new lock override mechanism (https://github.com/IQSS/dataverse/pull/7568).
* Do a deposit manually.
* Create a second version manually.
* Do an import with `dd-dans-deposit-to-dataverse`, including update deposits.
* Observe that the vault metadata is set correctly and that that dataset does not stay locked.

# Related PRs 
https://github.com/IQSS/dataverse/pull/7568

# Notify
@DANS-KNAW/dataversedans
